spec/rails_helper.rb

```diff
@@ -1,5 +1,10 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV["RAILS_ENV"] ||= 'test'
+
+# for reCAPTCHA testing
+ENV["RECAPTCHA_PRIVATE_KEY"] = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```

Comment:
These keys are for testing(?), not the real keys. Real keys go into Heroku Settings, should these be taken out?